srcpkgs/llvm17/template

```diff
@@ -629,31 +653,40 @@ libcxx-devel_package() {
 	}
 }
 
+compiler-rt_package() {
+	short_desc+=" - runtime libraries"
+	homepage="https://compiler-rt.llvm.org/"
+	pkg_install() {
+		vmove usr/lib/clang/17/lib
+		vmove usr/lib/clang/17/bin
```

Comment:
Forgot this also needs `usr/lib/clang/17/share`, those files currently end up in `llvm17`;
```
/usr/lib/clang/17/share/asan_ignorelist.txt
/usr/lib/clang/17/share/cfi_ignorelist.txt
/usr/lib/clang/17/share/dfsan_abilist.txt
/usr/lib/clang/17/share/hwasan_ignorelist.txt
/usr/lib/clang/17/share/msan_ignorelist.txt
```

Reply:
`usr/lib/clang/17/share` and `usr/lib/clang/17/bin` are not built on armv*, along with many of the components in `usr/lib/clang/17/bin`